.eslintrc.json

```diff
@@ -0,0 +1,25 @@
+{
+  "env": {
+    "browser": true,
+    "es6": true,
+    "jest": true
+  },
+  "parser": "babel-eslint",
+  "parserOptions": {
+    "ecmaVersion": 2018,
+    "sourceType": "module"
+  },
+  "extends": ["airbnb-base"],
+  "rules": {
+    "no-shadow": "off",
+    "no-param-reassign": "off",
+    "eol-last": "off",
+    "import/extensions": [ 1, {
+      "js": "always", "json": "always"
+    }]
```

Comment:
- [ ] It would be better if we use the Microverse recommended linter file as in the link [here](https://github.com/microverseinc/linters-config/blob/master/javascript/.eslintrc.json). This is used by all the students of Microverse to enforce a coding standard

Reply:
I used [this](https://github.com/microverseinc/linters-config/blob/master/html-css-js/.eslintrc.json) linters provided by Microverse for projects which include HTML+CSS+JS. The one you shared with me is for projects which only use JS. But let me know if for some reason it's better to use that one. Thank you!

Reply:
Thank you for sharing the link!!!!